its/ruling/src/test/resources/expected/python-S116.json

```diff
@@ -526,7 +526,6 @@
 'project:twisted-12.1.0/twisted/internet/task.py':[
 63,
 64,
-112,
```

Comment:
I don't understand why this issue is missing and not relevant anymore : this is an instance field and should be detected as such.

Reply:
And I figured it out : this self variable is instantiated via a class method. This is way out of scope of this sprint, so fine to accept this "loss" for now.